Scripts/launch/poseEstimator.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+
+	<include file="./imuDvl.launch" />
```

Comment:
@prudhvid Sure.

Can you tell me what `.` refers to? (I believe it refers to the directory that the `poseEstimator.launch` file is in?)